src/utilities/permissions.ts

```diff
@@ -405,7 +411,7 @@ const featurePermissions: FeaturePermissions = {
     canCreate: user => isUserAdmin(user) || queryPermissions.CREATE_PLAN(user),
     canDelete: (user, plan) => isUserAdmin(user) || (isPlanOwner(user, plan) && queryPermissions.DELETE_PLAN(user)),
     canRead: user => isUserAdmin(user) || queryPermissions.GET_PLAN(user),
-    canUpdate: () => false, // no feature to update plans exists
+    canUpdate: (user, plan) => isUserAdmin(user) || (isPlanOwner(user, plan) && queryPermissions.DELETE_PLAN(user)),
```

Comment:
should this permission be using the `DELETE_PLAN` query?

Reply:
Whoops good catch